templates/python/custom-copilot-basic/env/.env.dev.user.tpl

```diff
@@ -3,11 +3,25 @@
 # Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
 SECRET_BOT_PASSWORD=
 {{#useOpenAI}}
-SECRET_OPENAI_API_KEY=<openai-api-key>
-SECRET_OPENAI_MODEL_DEPLOYMENT_NAME=<openai-model-development-name>
+{{#openAIKey}}
+SECRET_OPENAI_API_KEY='{{{openAIKey}}}'
+{{/openAIKey}}
+{{^openAIKey}}
+SECRET_OPENAI_API_KEY=
+{{/openAIKey}}
 {{/useOpenAI}}
 {{#useAzureOpenAI}}
-SECRET_AZURE_OPENAI_API_KEY=<azure-openai-api-key>
-SECRET_AZURE_OPENAI_MODEL_DEPLOYMENT_NAME=<azure-openai-model-development-name>
-SECRET_AZURE_OPENAI_ENDPOINT=<azure-openai-endpoint>
+{{#azureOpenAIKey}}
+SECRET_AZURE_OPENAI_API_KEY='{{{azureOpenAIKey}}}'
+{{/azureOpenAIKey}}
+{{^azureOpenAIKey}}
+SECRET_AZURE_OPENAI_API_KEY=
+{{/azureOpenAIKey}}
+SECRET_AZURE_OPENAI_MODEL_DEPLOYMENT_NAME=
```

Comment:
What's the conclusion  of the naming? Do we prefer `SECRET_AZURE_OPENAI_MODEL_DEPLOYMENT_NAME` or `AZURE_OPENAI_MODEL_DEPLOYMENT_NAME`?

Reply:
@MuyangAmigo what is your idea?

Reply:
I think `AZURE_OPENAI_MODEL_DEPLOYMENT_NAME` is enough.